libgalois/include/katana/PropertyGraph.h

```diff
@@ -196,39 +196,9 @@ class KATANA_EXPORT PropertyGraph {
   static Result<std::unique_ptr<PropertyGraph>> Make(
       std::unique_ptr<tsuba::RDGFile> rdg_file, tsuba::RDG&& rdg);
 
-  /// Make a property graph from an RDG name. Load the default partition and all
-  /// properties.
+  /// Make a property graph from an RDG name.
   static Result<std::unique_ptr<PropertyGraph>> Make(
-      const std::string& rdg_name);
-
-  /// Make a property graph from an RDG name. Load the requested partition and
-  /// all properties.
-  ///
-  /// This is probably not the Make you are looking for. This is used by the
-  /// partitioner, which makes its living manipulating graph representation
-  /// internals.
-  static Result<std::unique_ptr<PropertyGraph>> Make(
-      const std::string& rdg_name, uint32_t host_to_load);
-
-  /// Make a property graph from an RDG name. Load the default partition and the
-  /// named node and edge properties.
-  static Result<std::unique_ptr<PropertyGraph>> Make(
-      const std::string& rdg_name,
-      const std::vector<std::string>& node_properties,
-      const std::vector<std::string>& edge_properties);
-
-  /// Full strength PropertyGraph::Make. Make a property graph from an RDG name.
-  /// Optionally specify a partition to load. Optionally specify properties to
-  /// load. If node_properties is null, all node properties will be
-  /// loaded. Likewise for edge_properties.
-  ///
-  /// This is probably not the Make you are looking for. This is used by the
-  /// partitioner, which makes its living manipulating graph representation
-  /// internals.
-  static Result<std::unique_ptr<PropertyGraph>> Make(
-      const std::string& rdg_name, std::optional<uint32_t> host_to_load,
-      const std::vector<std::string>* node_properties,
-      const std::vector<std::string>* edge_properties);
+      const std::string& rdg_name, const tsuba::RDGLoadOptions& opts);
```

Comment:
Maybe `..., const tsuba::RDGLoadOptions& opts = tsuba::RDGLoadOptions());`? Totally a personal preference thing feel free to ignore.

Reply:
I would prefer having the default. The reason is that it feels like a leaky abstraction for users of `katana::` to need to also use `tsuba::` in the standard code path. Having a default argument will allow users to ignore `tsuba::` in the simple case.